prometheus/metrics.go

```diff
@@ -67,12 +66,6 @@ type Metric struct {
 	err    error
 }
 
-// MetricsVector holds the Prometheus Vector model, which contains a sample from one or more time series
```

Comment:
Is this type moved or just deleted ?

Reply:
It's deleted. The actual workflow has been: modified, remodified, remodified, deleted :)
But I don't really care if you think it's better to have it back, I can put it back. It just wraps the prometheus model, I wasn't sure it was necessary.